src/main.c

```diff
@@ -0,0 +1,118 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include "../include/bmi.h"
+#include "../include/et.h"
+#include "../include/bmi_et.h"
+
+#include "../../AORC/include/aorc.h"
```

Comment:
these two AORC paths need to be updated

Reply:
Updated. But that main.c file might not be needed...